types/instance.d.ts

```diff
@@ -59,6 +59,9 @@ export interface FastifyInstance<
 
   register: FastifyRegister<FastifyInstance<RawServer, RawRequest, RawReply, Logger> & PromiseLike<undefined>>;
 
+  getDefaultRoute: DefaultRoute<RawRequest, RawReply>;
```

Comment:
I think these should still point to one of the existing route types. This new `DefaultRoute` type is missing the fastify request and reply types

Reply:
I still think that we should remove the global handler stuff from `fastify-websocket` that necessitates these additions. Does anyone super object to that?

Reply:
@Ethan-Arrowood Since the `defaultRoute` is the fallback function that is used when no route match is found, it always takes nodeJS core req/res objects:
https://github.com/fastify/fastify/blob/master/fastify.js#L526

Like @airhorns mentioned, this addition can be eliminated if no one objects to removing the usage of the global handler. A good case for that was made here: https://github.com/fastify/fastify-websocket/pull/95#discussion_r541967028